composer.json

```diff
@@ -1,15 +1,26 @@
 {
     "name": "chtombleson/silverstripe-flowchart",
     "description": "Flowchart silverstripe module.",
-    "type": "silverstripe-module",
+    "type": "silverstripe-vendormodule",
     "keywords": ["silverstripe"],
-    "minimum-stability": "dev",
     "license": "BSD-3-Clause",
     "require": {
-        "silverstripe/cms": "~3.1",
-        "silverstripe/framework": "~3.1"
+        "silverstripe/cms": "^4",
```

Comment:
Does it matter if this isn't the latest version of SS4? I.e. should it be `"^4.2"` ?

Reply:
It means the latest of `4.x`